src/java.base/share/classes/java/lang/CharacterDataLatin1.java.template

```diff
@@ -135,30 +135,36 @@ class CharacterDataLatin1 extends CharacterData {
     }
 
     int toLowerCase(int ch) {
-        int mapChar = ch;
-        int val = getProperties(ch);
-
-        if (((val & $$maskLowerCase) != 0) && 
-                ((val & $$maskCaseOffset) != $$maskCaseOffset)) { 
-            int offset = val << $$shiftCaseOffsetSign >> ($$shiftCaseOffsetSign+$$shiftCaseOffset);
-            mapChar = ch + offset;
+        if (ch < 'A') { // Fast path for low code points
+            return ch;
         }
-        return mapChar;
+        int l = ch | 0x20; // Lowercase using 'oldest ASCII trick in the book'
+        if ( l <= 'z' // In range a-z
```

Comment:
```suggestion
        if (l <= 'z' // In range a-z
```

Reply:
Fixed! (My IDE does not highlight this code, making it a bit harder to spot mistakes like this)